### PR TITLE
Fix selection of language chunk

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

As internally discussed.

### CODE REVIEW

File: packages/l10n/language-information.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ import extractLanguageFromLocale from './utils/extract-language-from-locale';
 const getImportChunk = lang => {
   switch (lang) {
     case 'de':
-      return import(/* webpackChunkName: "language-data-es" */ './data/languages/es.json');
+      return import(/* webpackChunkName: "language-data-es" */ './data/languages/de.json');
```
Comment (montezume) [inline packages/l10n/language-information.js]: 🙈

Comment (tdeekens) [reply packages/l10n/language-information.js]: It confused the living hell out of me. You go to German and get Spanish and vice versa. Like what's going on 😄. Good to "hear" from you :)

Comment (qmateub) [reply packages/l10n/language-information.js]: We call it easter eggs :trollface:

Comment (montezume) [reply packages/l10n/language-information.js]: @tdeekens can you change the `webpackChunkName` as well?

**Pull request merged**